lnd_test.go

```diff
@@ -110,7 +117,9 @@ func assertTxInBlock(t *harnessTest, block *wire.MsgBlock, txid *chainhash.Hash)
 
 // mineBlocks mine 'num' of blocks and check that blocks are present in
 // node blockchain.
-func mineBlocks(t *harnessTest, net *networkHarness, num uint32) []*wire.MsgBlock {
+func mineBlocks(t *harnessTest, net *lntest.NetworkHarness, num uint32,
+) []*wire.MsgBlock {
```

Comment:
Odd line wrapping here. Not replicated in the existing codebase.

Reply:
How would you format this declaration to not violate the 80 char limit?